docs/components/modeler/web-modeler/import-diagram.md

```diff
@@ -0,0 +1,31 @@
+---
+id: import-diagram
+title: Import diagram
+description: "You can import a BPMN diagram at any time with Web Modeler."
+---
+
+You can import a BPMN diagram at any time with Web Modeler.
+
+There are different ways how you can achieve that:
+
+1. In a Project, click **New** and choose **Upload Files** and then select the files from your machine
```

Comment:
Why are resources ("Diagram", "Project") on this page capitalized while on the other pages (e.g. `collaboration.md`) they are not?

Reply:
For additional context, there are some discrepancies between the design team and the technical writing style guide. The design team chooses to capitalize items like "Diagram" and "Project" where we do not.